micro_brewery/data/uom_uom.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo noupdate="1">
+    <record id="uom_uom_27" model="uom.uom">
+        <field name="name">hL</field>
+        <field name="category_id" ref="uom.product_uom_categ_vol"/>
+        <field name="factor_inv">100</field>
```

Comment:
This one could have stayed, because it is accurate (but no need to change it again)